worldedit-bukkit/adapters/adapter-1_19/src/main/java/com/sk89q/worldedit/bukkit/adapter/impl/fawe/v1_19_R1/PaperweightGetBlocks.java

```diff
@@ -1117,6 +1115,16 @@ public boolean hasSection(int layer) {
         return getSections(false)[layer] != null;
     }
 
+    @Override
+    public int[] loadInts(final int layer) {
+        return null;
+    }
+
+    @Override
+    public int[] loadIntsIfPresent(final int layer) {
```

Comment:
I would move these methods into a superclass so they're not needlessly duplicated in each of the "final" classes (i.e. into CharGetBlocks)

Reply:
Done, and I have also had them throw an exception since they shouldn't be called on char based platforms.